protoc-gen-hack/plugin.go

```diff
@@ -324,7 +317,16 @@ func writeFileDescriptor(w *writer, fdp *desc.FileDescriptorProto) {
 	w.p("}")
 	w.ln()
 	w.p("public function FileDescriptorProtoBytes(): string {")
-	w.p("return (string)\\gzuncompress(\\file_get_contents(\\realpath(\\dirname(__FILE__)) . '/%s'));", filepath.Base(binaryFileDescriptorPath(fdp)))
+	// This logic matches what protoc-gen-go plugin does. See this SO thread for an example:
+	// https://stackoverflow.com/questions/60540511/how-to-display-protoc-gen-go-gzipped-filedescriptorproto-as-plaintext
+	binaryProto := generateBinaryFileDescriptor(fdp)
+	binaryProtoStr := ""
+	for i := 0; i < len(binaryProto); i++ {
+		c := binaryProto[i]
+		binaryProtoStr += fmt.Sprintf("\\x%x", c)
```

Comment:
`%x` prints a single byte in hexadecimal (perfect for the `\x??` format in Hacklang)